servers/curated-corpus-api/schema-admin.graphql

```diff
@@ -1217,6 +1217,20 @@ input DeleteScheduledCorpusItemInput {
   actionScreen: ActionScreen
 }
 
+type UrlMetadata {
+  url: String! @override(from: "prospect-api")
+  imageUrl: String @override(from: "prospect-api")
+  publisher: String @override(from: "prospect-api")
+  datePublished: String @override(from: "prospect-api")
+  domain: String @override(from: "prospect-api")
+  title: String @override(from: "prospect-api")
+  excerpt: String @override(from: "prospect-api")
+  language: String @override(from: "prospect-api")
+  isSyndicated: Boolean @override(from: "prospect-api")
+  isCollection: Boolean @override(from: "prospect-api")
+  authors: String @override(from: "prospect-api")
+}
```

Comment:
I remember watching an Apollo video on `@override` years ago. Could you refresh my memory about what this does, and why we should use it here?